src/shims/ApiCompat.proj

```diff
@@ -26,6 +26,13 @@
     <_RunApiCompat Condition="'$(BuildAllConfigurations)' == 'true' and $(TargetGroup.StartsWith('uap'))">false</_RunApiCompat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- Run API compat against the following 1:1 assemblies but don't include them in the list that is used to generate facades -->
+    <NetFxReference Include="System.DirectoryServices" />
+    <NetFxReference Include="System.DirectoryServices.AccountManagement" />
+    <NetFxReference Include="System.DirectoryServices.Protocols" />
+  </ItemGroup>
+
   <!-- Run ApiCompat -->
   <Target Name="RunApiCompat"
           Condition="'$(_RunApiCompat)' == 'true'"
```

Comment:
@danmosemsft what do you think? You think we are at a point where we can just transfer everything to the baseline file and turn this into an error if it breaks? I was hesitant in the past because of all the auto-update PR's that might break this but I think we are past that point for the most part now.